lstmcpipe/onsite_mc_r0_to_dl1.py

```diff
@@ -136,14 +145,39 @@ def main(input_dir, config_file=None, train_test_ratio=0.5, random_seed=42, n_r0
         print(f"\n ==== START {os.path.basename(__file__)} ==== \n")
         # This formatting should be the same as in `onsite_mc_r0_to_dl3.py`
         today = calendar.datetime.date.today()
-        base_prod_id = f'{today.year:04d}{today.month:02d}{today.day:02d}_v{lstchain.__version__}'
+        if workflow_kind == 'lstchain':
+            base_prod_id = f'{today.year:04d}{today.month:02d}{today.day:02d}_v{lstchain.__version__}'
+        elif workflow_kind == 'ctapipe':
+            import ctapipe
+            base_prod_id = f'{today.year:04d}{today.month:02d}{today.day:02d}_v{ctapipe.__version__}'        
```

Comment:
Naive comment. Should we (do you want to) differentiate the prod_ids run with ctapipe from the ones run with lstchain ? In the RTA case we add a `_vRTA` suffix. But as you prefer :)

Reply:
I have no opinion on this. My approach would have been to collect them in different folders:
- DL1_lstchain
- DL1_hiperta
- DL1_ctapipe

But there might be reasons against this, that I dont think of right now

Reply:
This would cause 3 cases to manage. 
I would suggest to leave it as it is - (note that this will only happen `if not flag_full_workflow`). And most probable it will disappear soon - working in a refactoring of the code.